.github/workflows/dotnet-format.yml

```diff
@@ -0,0 +1,26 @@
+name: dotnet format
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+    - '**.cs'
```

Comment:
I guess we need to add `.editorconfig` as well in case the rule change caused failure.

Reply:
done